daml-lf/spec/transaction.rst

```diff
@@ -428,21 +455,16 @@ Every element of ``actors``, ``stakeholders``, ``signatories``, and
   The ``stakeholders`` and ``signatories`` field have the same meaning
   they have for ``NodeCreate``.
 
-* `return_value` is required
+  The ``actors`` field contains the parties that exercised the choice.
+  The ``controllers`` field contains the parties that _can_ exercise
+  the choice. Note that according to the ledger model these two fields
+  _must_ be the same. For this reason the ``controllers`` field was
+  removed in version 6 -- see *since version 10* below.
```

Comment:
Given that we don't support version 6 anymore can we simply drop the `controllers` field and remove the sentence?

Reply:
dropped.